enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/AuthProceduresTest.java

```diff
@@ -544,6 +547,72 @@ public void userDeletion4() throws Exception
         }
     }
 
+    //----------User suspension scenarios-----------
```

Comment:
Unit tests are missing for activating and suspending users from the _procedures_ point of view. At least the following tests need to be added:

Activating an already activated user
Suspending an already suspended user
Non-admin activating/suspending other user